src/CMakeLists.txt

```diff
@@ -54,6 +60,13 @@ qt_add_qml_module(${PROJECT_NAME}
         QtQuick.Controls.Basic
 )
 
+list(APPEND QML_DIRS "/usr/lib64/qt6/qml")
+#TODO check QML output directory
+#list(APPEND QML_DIRS "${PROJECT_SOURCE_DIR}/src")
+list(REMOVE_DUPLICATES QML_DIRS)
+set(QML_IMPORT_PATH "${QML_DIRS}" CACHE STRING "Add QML imports" FORCE)
+set(QML_IMPORT_TRACE true)
```

Comment:
What's the function of this variable? Better to leave a comment

Reply:
It is not needed anymore, but JIC I was trying to append QML modules directories